common/djangoapps/enrollment/urls.py

```diff
@@ -3,28 +3,18 @@
 
 """
 from django.conf import settings
-from django.conf.urls import patterns, url
+from django.conf.urls import url
 
 from .views import EnrollmentCourseDetailView, EnrollmentListView, EnrollmentView
 
-urlpatterns = patterns(
-    'enrollment.views',
-    url(
-        r'^enrollment/{username},{course_key}$'.format(
-            username=settings.USERNAME_PATTERN, course_key=settings.COURSE_ID_PATTERN
-        ),
-        EnrollmentView.as_view(),
-        name='courseenrollment'
-    ),
-    url(
-        r'^enrollment/{course_key}$'.format(course_key=settings.COURSE_ID_PATTERN),
-        EnrollmentView.as_view(),
-        name='courseenrollment'
-    ),
+urlpatterns = [
+    url(r'^enrollment/{username},{course_key}$'.format(
+        username=settings.USERNAME_PATTERN,
+        course_key=settings.COURSE_ID_PATTERN),
+        EnrollmentView.as_view(), name='courseenrollment'),
```

Comment:
nit: the indentation seems a little confusing here (format arguments not indented)

Reply:
I agree, and tried it a few different ways but this was the only one that made it past linting. It didn't seem worth continuing to fight over.